src/main/kotlin/sg/com/quantai/etl/controllers/CryptoController.kt

```diff
@@ -0,0 +1,30 @@
+package sg.com.quantai.etl.controllers
+
+import com.fasterxml.jackson.databind.JsonNode
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import sg.com.quantai.etl.services.CryptoService
+
+@RestController
+class CryptoController(private val cryptoService: CryptoService) {
+
+    // Endpoint to fetch current crypto price
+    @GetMapping("/crypto/current/price")
```

Comment:
Make sure that the endpoint follows the same convention (HTTP Method + endpoint URL) adopted to the news article data types, described at docs/NewsArticles.md

Reply:
Done